src/Promitor.Integrations.Sinks.OpenTelemetry/Collectors/OpenTelemetrySystemMetricsSink.cs

```diff
@@ -20,5 +20,10 @@ public async Task WriteGaugeMeasurementAsync(string name, string description, do
         {
             await _metricSink.ReportMetricAsync(name, description, value, labels);
         }
+
+        public Task WriteHistogramMeasurementAsync(string name, string description, double value, Dictionary<string, string> labels, bool includeTimestamp)
+        {
+            throw new System.NotImplementedException("Histogram measurement not yet supported in OpenTelemetry sink");
```

Comment:
Let's not throw but log or ignore instead; this would break end-users using this sink